config/initializers/event_bus.rb

```diff
@@ -22,7 +22,8 @@
 
   # send bulk emails after events
   Event::BULK_MAIL_KINDS.each do |kind|
-    config.listen("#{kind}_event") { |event| SendBulkEmailJob.perform_later(event.id) }
+    delay = ENV.fetch('BULK_MAIL_DELAY', 0).to_i.seconds
+    config.listen("#{kind}_event") { |event| SendBulkEmailJob.set(wait: delay).perform_later(event.id) }
```

Comment:
actually.. why don' we pass in the event rather than the event_id? then we would not have to have this fuzzy fail thing...right?